snakemake/executors/ga4gh_tes.py

```diff
@@ -329,12 +330,16 @@ def _get_task(self, job, jobscript):
         task["resources"] = tes.models.Resources()
 
         # define resources
-        if "_cores" in job.resources:
-            task["resources"]["cpu_cores"] = job.resources["_cores"]
-        if "mem_mb" in job.resources:
-            task["resources"]["ram_gb"] = job.resources["mem_mb"] / 1000
-        if "disk_mb" in job.resources:
-            task["resources"]["disk_gb"] = job.resources["disk_mb"] / 1000
+        if job.resources.get("_cores"):
+            setattr(task["resources"], "cpu_cores", job.resources["_cores"])
+        if job.resources.get("mem_mb"):
+            setattr(
+                task["resources"], "ram_gb", math.ceil(job.resources["mem_mb"] / 1000)
+            )
```

Comment:
Can't you just write the following (and the same below)?

```suggestion
            task["resources"].ram_gb = math.ceil(job.resources["mem_mb"] / 1000
```

Reply:
no, it does not compile then, it has to be via setattr. at least after some google search.

Reply:
Shouldn't this rather be:

```python
task.resources.cpu_cores = job.resources["_cores"]
```

...and so on?